src/behavior-considered-undefined.md

```diff
@@ -70,6 +72,13 @@ Please read the [Rustonomicon] before writing unsafe code.
   some allocated object as a non-pointer type (such as integers).
   'Reinterpreting' refers to loading the pointer value at integer type without a
   cast, e.g. by doing raw pointer casts or using a union.
+* Violating assumptions of the Rust runtime. This is only possible using
+  mechanisms outside Rust. Most assumptions of the Rust runtime are currently
+  not explicitly documented.
+  * For assumptions specifically related to unwinding, see [unwinding-ub].
+  * Deallocating a Rust stack frame without executing destructors for local
+    variables owned by the stack frame is always undefined. This can occur with
+    C functions like `longjmp`.
```

Comment:
```suggestion
  * The runtime assumes that a Rust stack frame is not deallocated without
    executing destructors for local variables owned by the stack frame. This can occur with
    C functions like `longjmp`.
```

Reply:
Ah, thanks, that's better; but it leaves the antecedent of "this" in the next sentence unclear, so I'm tweaking it further.

```suggestion
  * The runtime assumes that a Rust stack frame is not deallocated without
    executing destructors for local variables owned by the stack frame. This assumption
    can be violated by C functions like `longjmp`.
```